docs/content/concepts/transactions/transaction-lifecycle.mdx

```diff
@@ -1,6 +1,27 @@
 ---
 title: Transaction Lifecycle
+description: The lifecycle of a transaction on the Sui network has some differences compared to those from other blockchains. 
 draft: true
 ---
 
-Content coming soon
\ No newline at end of file
+At a high level, the following figure outlines the lifecycle of a transaction on the Sui blockchain.
+
+![Transaction lifecycle](./images/txn-lifecycle.png)
+
+The following steps align with those in the preceding image. 
+
+1. The first step of the process is the creation of a transaction. A user with a private key creates and signs a user transaction to either mutate objects they own, or a mix of objects they own and [shared objects](../object-ownership/shared.mdx).
+
+1. Sui sends the transaction to each validator (often through a Full node). Validators perform a series of validity and safety checks, sign it, and return the signed transaction to the client. 
```

Comment:
The numbering of the paragraphs does not seem to align with the future (it is alway (1))

Reply:
Markdown automagically assigns correct numbers on output (regardless of the number actually used).